src/libraries/Native/native-binplace.proj

```diff
@@ -10,7 +10,7 @@
 
   <!-- Ordering matters! Overriding GetBinPlaceItems and Build targets after the Sdk import. -->
   <Target Name="GetBinPlaceItems">
-    <ItemGroup>
+    <ItemGroup Condition="'$(BinPlaceTestRuntimePack)' == ''">
```

Comment:
This boolean-emulating property should only have two states to follow good MSBuild practice:

```suggestion
    <ItemGroup Condition="'$(BinPlaceTestRuntimePack)' != 'true'">
```